agent/lib/src/external_api/model.rs

```diff
@@ -0,0 +1,139 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Deserialize, Serialize, Debug, PartialEq, Eq)]
+pub struct FileModel {
+    pub filename: String,
+    pub content: String,
+}
+
+#[derive(Deserialize, Serialize, Debug)]
+pub struct CodeEntry {
+    pub files: Vec<FileModel>,
+    pub script: Vec<String>, // All commands to execute at startup
+}
```

Comment:
Is `CodeEntry` the entry point for running some piece of code?

Reply:
This is a legacy piece of code that is not used anymore, replaced by ``RequestData``. Thank you for noticing it